data/presets/presets.json

```diff
@@ -3317,6 +3317,295 @@
+            ],
+            "geometry": [
+                "point",
+                "area"
+            ],
+            "terms": [
+                "theme park",
+                "roundabout",
+                "merry-go-round",
+                "galloper",
+                "jumper",
+                "horseabout",
+                "flying horses"
+            ],
+            "tags": {
+                "attraction": "carousel"
+            },
+            "name": "Carousel"
+        },
+        "attraction/dark_ride": {
+            "icon": "rail-metro",
+            "fields": [
+                "name",
+                "operator",
+                "opening_hours"
+            ],
+            "geometry": [
+                "point",
+                "line",
+                "area"
+            ],
+            "terms": [
+                "theme park",
+                "ghost train"
+            ],
+            "tags": {
+                "attraction": "dark_ride"
+            },
+            "name": "Dark Ride"
+        },
+        "attraction/drop_tower": {
+            "icon": "poi-tower",
+            "fields": [
+                "name",
+                "operator",
+                "height",
+                "opening_hours"
+            ],
+            "geometry": [
+                "point",
+                "area"
+            ],
+            "terms": [
+                "theme park",
+                "amusement ride",
+                "gondola",
+                "tower",
+                "big drop"
+            ],
+            "tags": {
+                "attraction": "drop_tower"
+            },
+            "name": "Drop Tower"
+        },
+        "attraction/pirate_ship": {
+            "icon": "amusement-park",
+            "fields": [
+                "name",
+                "operator",
+                "opening_hours"
+            ],
+            "geometry": [
+                "point"
+            ],
+            "terms": [
+                "theme park",
+                "carnival ride",
+                "amusement ride"
+            ],
+            "tags": {
+                "attraction": "pirate_ship"
+            },
+            "name": "Pirate Ship"
+        },
+        "attraction/river_rafting": {
+            "icon": "ferry",
+            "fields": [
+                "name",
+                "operator",
+                "opening_hours"
+            ],
+            "geometry": [
+                "point",
+                "line"
+            ],
+            "terms": [
+                "theme park",
+                "aquatic park",
+                "water park",
+                "rafting simulator",
+                "river rafting ride",
+                "river rapids ride"
+            ],
+            "tags": {
+                "attraction": "river_rafting"
+            },
+            "name": "River Rafting"
+        },
+        "attraction/roller_coaster": {
+            "icon": "amusement-park",
+            "fields": [
```

Comment:
Would it also make sense to add `fee` onto these presets? Most parks charge per ride.